terraform/modules/firewall-policy/main.tf

```diff
@@ -49,6 +49,21 @@ resource "aws_networkfirewall_rule_group" "stateful" {
   }
 }
 
+resource "aws_networkfirewall_rule_group" "fqdn-stateful" {
+  capacity = 100
```

Comment:
The `capacity` value seems low to me. Should this be supplied with a variable, with a greater default value? The documented maximum for a rule group is `30000`, but the implied limit for a policy is `30000`, and that would be a combination of all the associated rule groups.